plasmapy/formulary/tests/test_collisions.py

```diff
@@ -1698,3 +1700,347 @@ def test_kwarg_method_error(self):
+
+        T_a = (frequencies.T_a * k_B).to(u.eV).value
+        T_b = (frequencies.T_b * k_B).to(u.eV).value
+
+        limit_values = []
+
+        if interaction_type == "e|e":
+            if limit_type == "slow":
+                limit_values.extend(
+                    [
+                        5.8 * 10**-6 * T_b ** (-3 / 2),
+                        5.8 * 10**-6 * T_b ** (-1 / 2) * T_a ** (-1),
+                        2.9 * 10**-6 * T_b ** (-1 / 2) * T_a ** (-1),
+                    ]
+                )
+            elif limit_type == "fast":
+                limit_values.extend(
+                    [
+                        7.7 * 10**-6 * T_a ** (-3 / 2),
+                        7.7 * 10**-6 * T_a ** (-3 / 2),
+                        3.9 * 10**-6 * T_b * T_a ** (-5 / 2),
+                    ]
+                )
+        elif interaction_type == "e|i":
+            mu = (frequencies.field_particle.mass / m_p).value
+
+            if limit_type == "slow":
+                limit_values.extend(
+                    [
+                        0.23 * mu ** (3 / 2) * T_b ** (-3 / 2),
+                        2.5 * 10**-4 * mu ** (1 / 2) * T_b ** (-1 / 2) * T_a ** (-1),
+                        1.2 * 10**-4 * mu ** (1 / 2) * T_b ** (-1 / 2) * T_a ** (-1),
+                    ]
+                )
+            elif limit_type == "fast":
+                limit_values.extend(
+                    [
+                        3.9 * 10**-6 * T_a ** (-3 / 2),
+                        7.7 * 10**-6 * T_a ** (-3 / 2),
+                        2.1 * 10**-9 * mu ** (-1) * T_b * T_a ** (-5 / 2),
+                    ]
+                )
+        elif interaction_type == "i|e":
+            mu = (frequencies.test_particle.mass / m_p).value
+
+            if limit_type == "slow":
+                limit_values.extend(
+                    [
+                        1.6 * 10**-9 * mu ** (-1) * T_b ** (-3 / 2),
+                        3.2 * 10**-9 * mu ** (-1) * T_b ** (-1 / 2) * T_a ** (-1),
+                        1.6 * 10**-9 * mu ** (-1) * T_b ** (-1 / 2) * T_a ** (-1),
+                    ]
+                )
+            elif limit_type == "fast":
+                limit_values.extend(
+                    [
+                        1.7 * 10**-4 * mu ** (1 / 2) * T_a ** (-3 / 2),
+                        1.8 * 10**-7 * mu ** (-1 / 2) * T_a ** (-3 / 2),
+                        1.7 * 10**-4 * mu ** (1 / 2) * T_b * T_a ** (-5 / 2),
+                    ]
+                )
+
+        elif interaction_type == "i|i":
+            mu = (frequencies.test_particle.mass / m_p).value
+            mu_prime = (frequencies.field_particle.mass / m_p).value
+
+            if limit_type == "slow":
+                limit_values.extend(
+                    [
+                        6.8
+                        * 10**-8
+                        * mu_prime ** (1 / 2)
+                        * mu ** (-1)
+                        * (1 + mu_prime / mu) ** (-1 / 2)
```

Comment:
I think this is a typo? 
```suggestion
                        * (1 + mu_prime / mu) 
```

Reply:
Great catch ! MIT has a typo in their formulary 👀
![image](https://user-images.githubusercontent.com/109694185/186032041-1c7558ba-fc4d-40ca-a9d5-c1a10d1718a8.png)

Reply:
Yikes - that's the 2013 formulary so I guess that got fixed in later versions! It's right in my 2016 one. 

If you want a digital copy here's a more up-to-date link I should have searched for when opening that issue...
https://library.psfc.mit.edu/catalog/online_pubs/NRL_FORMULARY_18.pdf